src/test/java/com/aws/greengrass/tes/CredentialRequestHandlerTest.java

```diff
@@ -78,8 +78,8 @@
     // Set in the far future so it doesn't expire
     private static final String EXPIRATION = "2030-08-19T07:35:15Z";
     private static final String RESPONSE_STR =
-            "{\"credentials\":" + "{\"accessKeyId\":\"" + ACCESS_KEY_ID + "\"," + "\"secretAccessKey\":\""
-                    + SECRET_ACCESS_KEY + "\"," + "\"sessionToken\":\"" + SESSION_TOKEN + "\"," + "\"expiration\":\""
+            "{\"credentials\":" + "{\"AccessKeyId\":\"" + ACCESS_KEY_ID + "\"," + "\"secretAccessKey\":\""
+                    + SECRET_ACCESS_KEY + "\"," + "\"sesSionToken\":\"" + SESSION_TOKEN + "\"," + "\"expiration\":\""
```

Comment:
why?

Reply:
Because V1 TES uses different capitalization, I wanted to ensure that we don't care how it is capitalized.